components/brave_rewards/resources/shared/components/onboarding/country_select.tsx

```diff
@@ -10,15 +10,25 @@ export function getCountryName (code: string) {
   return countryNames.of(code)
 }
 
-function getCountryOptions (countries: string[]) {
-  return countries
-    .map((code) => ({ code, name: countryNames.of(code) || '' }))
-    .filter((item) => Boolean(item.name))
+function getCountryOptions (countries: string[], defaultCountry: string) {
+  let options = countries
+    .map((code) => ({ code, name: countryNames.of(code) || '', bold: false }))
+    .filter((item) => item.code !== defaultCountry && Boolean(item.name))
     .sort((a, b) => a.name?.localeCompare(b.name))
+  
+  const defaultCountryName = countryNames.of(defaultCountry) || ''
+  if (countries.includes(defaultCountry) && Boolean(defaultCountryName)) {
```

Comment:
The `Boolean(...)` here isn't really necessary in this case since `defaultCountryName` will be coerced to boolean.

It wasn't necessary in the previous `filter` argument either, but I think I was just trying to be self-documenting there.